java/java.lsp.server/vscode/.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
```

Comment:
I noticed this in other JSON files aswell. Are these files from upstream somewhere or authored? This is not valid JSON (there is no comment in JSON (rfc7159)).

Reply:
This is what one gets when generating a skeletal VSCode project. Looks like these `.json` files accept comments, surprising - I might have used that instead of 2413e8b

Reply:
Ok - for me the important part is, that VSCode generates it so. I was just wondering why invalid JSON was created, but it seems web developers again ignored RFCs and invented a new format.